corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -221,31 +221,31 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                 </div>
               </div>
             </div>
-            {% if request|toggle_enabled:"ROLE_WEBAPPS_PERMISSIONS" %}
+            {% if request|toggle_enabled:"APP_ACCESS_PERMISSIONS" %}
               <div class="form-group">
                 <label class="col-sm-4 control-label">
-                  {% trans "View All Web Apps" %}
+                  {% trans "Access All Applications" %}
                 </label>
                 <div class="col-sm-8 controls">
                   <div class="form-check">
                     <input type="checkbox"
-                           id="access-all-webapps-checkbox"
+                           id="access-all-apps-checkbox"
                            data-bind="checked: webAppsPermissions.all, disable: !$root.allowEdit" />
```

Comment:
just checking if we could also rename things like `webAppsPermissions`. 

Also, if possible, I believe there are couple of things on the model too, like `view_web_apps` and `view_web_apps_list` which can be handled using a migration?

Reply:
I gave it some time. But it was not straight forward and I was unsure if changes name would result in breaking things. 
Maybe we can look it together when you have some time?

Reply:
Sure, lets offline